src/js/videojs-dash.js

```diff
@@ -35,6 +44,8 @@ class Html5DashJS {
     // again later once everything is setup
     tech.isReady_ = false;
 
+    tech.on('progress', cancelEvent);
```

Comment:
True, this will only block `progress` if the player is initially loaded with a DASH source, if you load a source later the other handlers are present (aka the one from videojs-errors I'm trying to work around)